README.md

```diff
@@ -22,16 +22,16 @@ or for use without any build tools:
 
 ```html
 <script type="module">
-  import { LightningAddress } from "https://esm.sh/@getalby/lightning-tools@5.0.0"; // jsdelivr.net, skypack.dev also work
-
-  // use LightningAddress normally...
-  (async () => {
-    const ln = new LightningAddress("hello@getalby.com");
-    // fetch the LNURL data
-    await ln.fetch();
-    // get the LNURL-pay data:
-    console.log(ln.lnurlpData);
-  })();
+	import { LightningAddress } from "https://esm.sh/@getalby/lightning-tools@5.0.0"; // jsdelivr.net, skypack.dev also work
```

Comment:
There is a lot of unrelated changes in this PR, and we normally don't have 4-space tabbing. Can you undo these?